src/DependencyInjection/EightPointsGuzzleExtension.php

```diff
@@ -91,13 +92,21 @@ public function load(array $configs, ContainerBuilder $container)
             // set service name based on client name
             $serviceName = sprintf('%s.client.%s', $this->getAlias(), $name);
             $container->setDefinition($serviceName, $client);
+
+            // Allowed only for Symfony 4.2+
+            if (\method_exists($container, 'registerAliasForArgument')) {
+                if ('%eight_points_guzzle_bundle.http_client.class%' !== $options['class']) {
+                    $container->registerAliasForArgument($serviceName, $options['class'], $name.'Client');
```

Comment:
Could you please add whitespace before and after dot? Like: `$var1 . $var2`. In all cases of concatenation.

Reply:
They where there, but Scrutinizer told me to remove them... Will add them again.